lib/db/query-trace-aggregator.js

```diff
@@ -74,7 +74,8 @@ class QueryTraceAggregator extends Aggregator {
         return
     }
 
-    if (segment.getDurationInMillis() < ttConfig.explain_threshold) {
+    const threshold = ttConfig.slow_query_threshold ?? ttConfig.explain_threshold
```

Comment:
If customers have already set `explain_threshold` this won't work since we default `slow_query_threshold` to 500.  This is different than the go agent configuration item which is `datastore_tracer.slow_query.threshold`.  I'm starting to think we don't address this since in our [docs website](https://docs.newrelic.com/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/#explain_threshold) the `transaction_tracer.explain_threshold` is pretty clear same for our comments in `lib/config.default.js`

>Minimum query duration (in milliseconds) for a transaction to be eligible for [slow queries](https://docs.newrelic.com/docs/apm/applications-menu/monitoring/viewing-slow-query-details/) in [transaction traces](https://docs.newrelic.com/docs/apm/transactions/transaction-traces/transaction-traces/).

Reply:
> If customers have already set `explain_threshold` this won't work since we default `slow_query_threshold` to 500.

🤦‍♂️ I just completely overlooked that.



> I'm starting to think we don't address this since

So, close the PR and issue?

Reply:
I think so. I don't see a world where we make this easier without breaking existing customers